components/badge/style.ts

```diff
@@ -1,37 +1,48 @@
 import { css } from "emotion";
-import { getColors, getFonts } from "../../shared/style";
+import { getColors, getFonts } from "../../shared/styles/core";
```

Comment:
I love that we have colors at one place under control 👍

Reply:
👍 However, we should not name them green, red, blue, dark-green, darker-green ... There are a lot of shades of green :-). Maybe we can use something like https://github.com/ryanzec/name-that-color .

Reply:
@Poltergeist this are the same name patterns used in DCOS UI/CNVAS and what the designers use

Reply:
Okay, maybe we keep that change for later.